tools/clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -7779,15 +7779,15 @@ def err_hlsl_unsupported_keyword_for_min_precision : Error<
    "%0 is only supported with -enable-16bit-types option">;
 def warn_hlsl_derivatives_in_wrong_shader_kind: Warning<
    "Intrinsic %0 potentially used by %1 requires derivatives - only available in pixel, compute, amplification, mesh, or broadcast node shaders.">,
-    DefaultError, InGroup<HLSLAvailability>;
+    InGroup<HLSLAvailability>;
 def warn_hlsl_derivatives_in_wrong_shader_model: Warning<
    "Intrinsic %0 potentially used by %1 requires derivatives - shader model 6.6 or greater is required for use in compute, amplification, or mesh shaders.">,
-    DefaultError, InGroup<HLSLAvailability>;
+    InGroup<HLSLAvailability>;
 def warn_hlsl_derivatives_wrong_numthreads : Warning<
    "Intrinsic %0 potentially used by %1 requires derivatives - when used in compute/amplification/mesh shaders"
     " or broadcast nodes, numthreads must be either 1D with X as a multiple of 4 or"
     " both X and Y must be multiples of 2.">,
-    DefaultError, InGroup<HLSLAvailability>;
+    InGroup<HLSLAvailability>;
 def warn_hlsl_intrinsic_overload_in_wrong_shader_model : Warning<
    "overload of intrinsic %0 requires shader model %1 or greater">, 
     DefaultError, InGroup<HLSLAvailability>;
```

Comment:
Do we also need to change this to a warning?

Reply:
Nice catch.
I'll keep it as error.

Reply:
Why was the `warn_hlsl_derivatives_wrong_numthreads` one undone?  This could impact existing usage on pre-SM 6.8.

I think @llvm-beanz question might have been misinterpreted.  Were you asking if `warn_hlsl_intrinsic_overload_in_wrong_shader_model` should also be changed to warning?  If so, we might need another group and warning to support the `DefaultError` case covering just the new SM 6.8 overload.  In fact, we should be doing all the checks for the new overload as well, right?  So it seems we need two groups of these, one that `DefaultError` and are only used for SM 6.8 new intinsics/overloads (only used when comparison sampler is used) and the other group used for pre-existing things.  Either that, or instead of removing `DefaultError`, we only diagnose them for the new SM 6.8 overload of the intrinsic, and do not use them for existing overloads.